dti-website/next.config.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
   webpack5: true,
+  experimental: { swcLoader: true },
+  typescript: { ignoreBuildErrors: true },
```

Comment:
does this just prevent type checking again?

Reply:
Yes, on CI type checking is already done in the check job, but next.js always run type checking again by default. This config prevents that.